flaml/automl.py

```diff
@@ -360,7 +361,7 @@ def classes_(self):
             return self._trained_estimator.classes_.tolist()
         return None
 
-    def predict(self, X_test):
+    def predict(self, X_test, freq=None):

```

Comment:
Add to doc string the new parameter `freq`. Perhaps it is better to make it part of `fit()`'s argument? In case in the future we want to do some frequency based optimization.

Reply:
got it, changes made locally

Reply:
`freq` was not used in `fit()`. I discussed this matter with Dr. Wang and he said not to add unnecessary things and to add `freq` as a part of `fit()` when needed